### PR TITLE
Milestone 1

### PR DESCRIPTION
**Pull request by jcy2704**
## Setup

Finished setting up the project.
- Removed unnecessary files
- Add react-redux
- Add redux
- Add prop-types
- Add linters
- Update README.md file

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
 import './App.css';
 
 function App() {
   return (
     <div className="App">
       <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
         <p>
-          Edit <code>src/App.js</code> and save to reload.
+          Edit
```
Comment (Forison) [inline src/App.js]: * [ ] Let us clear this component

File: src/index.css [hunk 2]
```diff
@@ -1,13 +1,27 @@
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
-    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
+  font-family:
```
Comment (Forison) [inline src/index.css]: * [ ] Let us clear this file

File: src/App.css [hunk 3]
```diff
@@ -32,6 +32,7 @@
   from {
```
Comment (Forison) [inline src/App.css]: * [ ] Let us clear this file

**Review by Forison — CHANGES_REQUESTED** [review]
# Status::Changes Required❌ 

# 👋  Chung

#### You have done a good job on this one so far, You have added all the required library. However, here is one issue we have to address to get this project approved😇.

* [ ] Let us remove all unwanted codes, at this state we need to remove all the unwanted boiler plates that are generated by `create-react-app` command.
> At this stage we should see 'hello world' when we start the app.

**[OPTIONAL]** 
* [ ] Let us make it a habit to add a description to all repo we create.

#### That is it, kindly make the change and submit a re-review👨‍🔧

#### Happy coding👨‍💻

Addo Forison

**Review by krys2fa — APPROVED** [review]
# Status: Project Approved! 🥇 
Congratulations on making all the requested changes and getting this milestone approved. 👍 

You can now merge this branch into the `development` branch and go on with the next milestone.

💡  Do not forget to submit your project completion form after merging this branch.

All the best,
Chris

**Pull request closed without merging**